_data/fellows.yml

```diff
@@ -228,3 +228,17 @@
 
     
 
+- name: Sai Pravallika Myneni
```

Comment:
@Pravallika-Myneni 's details are already there in the deployed website. This is repetitive. Please remove it

Reply:
removed the repeated details and kept the one with complete info @sumana2001